prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -75,8 +80,10 @@ class SftpTransport : ITransport, Logging {
     }
 
     companion object {
-        fun lookupCredentials(receiverFullName: String): SftpCredential {
-            val credentialLabel = receiverFullName
+
+        fun lookupCredentials(credentialName: String): SftpCredential {
+
+            val credentialLabel = credentialName
```

Comment:
Looks good, but a question on the extra spacing. Is this the default now for our linter? /cc @jimduff-usds

Reply:
ktlint haven't complied about it 

![image](https://user-images.githubusercontent.com/3750368/123569284-6f78f480-d79c-11eb-9d1a-d3d22808c3cc.png)

Reply:
@pablocaviglia, let's keep the spacing conformant with the way the rest of our code looks.